tools/go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/openshift/api/tools
 
-go 1.22.0
+go 1.23.0
 
-toolchain go1.22.1
+toolchain go1.23.4
```

Comment:
I think you can revert this, since the controller tools branch you're pulling is on go 1.22